### DISCUSSION TITLE
使用官方示例代码发送消息一直报错，各位大佬方便看下什么原因吗

### DISCUSSION THREAD

**Discussion by l12312331321231234** [apache/rocketmq#7843] [Q&A]
![1708493516468](https://github.com/apache/rocketmq/assets/95022064/2fa949ca-ea55-4d9e-85b3-3acc91758a24)
![1708493491896](https://github.com/apache/rocketmq/assets/95022064/1824e4e3-5afc-485b-8f74-5c703c55eaee)

**Comment by aaron-ai** [comment]
Did you start rocketmq proxy before message publishing?
